django/core/management/commands/inspectdb.py

```diff
@@ -123,6 +123,13 @@ def handle_inspection(self, options):
                     yield "# The error was: %s" % e
                     continue
 
+                def is_postgres_row(row):
+                    field_type, _, _ = self.get_field_type(connection, table_name, row)
+                    return field_type.startswith("postgres.fields.")
+
+                if any(is_postgres_row(row) for row in table_description):
+                    yield "from django.contrib import postgres"
```

Comment:
This is untested

Reply:
To be fair the other import is untested as well. But actually, this might be incorrect, let me check and fix.